documentation/src/docs/segment-store-service.md

```diff
@@ -130,17 +132,18 @@ The _Durable Log_ is the central component that handles all Log Operations. All
 1. All received operations are added to an _Operation Queue_ (the caller receives a Future which will be completed when the operation is durably persisted).
 2. The _Operation Processor_ picks all operations currently available in the queue (if the queue is empty, it will wait until at least one Operation is added).
 3. The _Operation Processor_ runs as a continuous loop (in a background thread), and has four main stages.
```

Comment:
It is mention in  3rd point that  `The _Operation Processor_ runs as a continuous loop (in a background thread), and has four main stages.` but only 3 points are mentioned below: please check

Reply:
then remove the number and say `executes the following steps`

Reply:
Accepted suggestion